docs/en/stack/security/securing-communications/tutorial-tls-addnodes.asciidoc

```diff
@@ -0,0 +1,189 @@
+which is used to discover the nodes in your cluster.
+
+For example, add the following settings to the `ES_PATH_CONF/elasticsearch.yml`
+file on the first node:
+
+[source,yaml]
+----
+xpack.security.enabled: true
+cluster.name: test-cluster
+node.name: node-1
+transport.tcp.port: 9301
+discovery.zen.ping.unicast.hosts: ["localhost:9302", "localhost:9303"]
+----
+
+Add the following settings to the `ES_PATH_CONF/elasticsearch.yml`
+file on the second node:
+
+[source,yaml]
+----
+xpack.security.enabled: true
+cluster.name: test-cluster
+node.name: node-2
+transport.tcp.port: 9302
+discovery.zen.ping.unicast.hosts: ["localhost:9301", "localhost:9303"]
+----
+
+Add the following settings to the `ES_PATH_CONF/elasticsearch.yml`
+file on the third node:
+
+[source,yaml]
+----
+xpack.security.enabled: true
+cluster.name: test-cluster
+node.name: node-3
+transport.tcp.port: 9303
+discovery.zen.ping.unicast.hosts: ["localhost:9301", "localhost:9302"]
+----
+
+TIP: In these examples, we have not specified the `transport.host`,
+`transport.bind_host`, or `transport.publish_host` settings, so they default to
+the `network.host` value. If you have not specified the `network.host` setting,
+it defaults to `_local_`, which represents the loopback addresses for the system. 
+
+If you choose different cluster names, node names, host names, or ports, you
+must substitute the appropriate values in subsequent steps as well. 
+--
+
+. Start each {es} node. For example, if you installed {es} with a `.tar.gz`
+package, run the following command from each {es} directory:
++
+--
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+./bin/elasticsearch
+----------------------------------------------------------------------
+
+See {ref}/starting-elasticsearch.html[Starting {es}].
+
+--
+
+. (Optional) Restart {kib}. For example, if you installed 
+{kib} with a `.tar.gz` package, run the following command from the {kib} 
+directory:
++
+--
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+./bin/kibana
+----------------------------------------------------------------------
+
+See {kibana-ref}/start-stop.html[Starting and stopping {kib}]. 
+--
+
+. Specify the minimum number of master-eligible nodes that must be available to
+form a cluster. 
++ 
+--
+By default, each node is eligible to be elected as the
+{ref}/modules-node.html#master-node[master node] and control the cluster. To
+avoid a _split brain_ scenario where multiple nodes elect themselves as the
+master, set the `discovery.zen.minimum_master_nodes` on each node in the cluster.
+
+For example, use the following API to change the
+`discovery.zen.minimum_master_nodes` setting:
+
+[source,js]
+----------------------------------
+PUT _cluster/settings
+{
+  "persistent": {
+    "discovery.zen.minimum_master_nodes": 2
+  }
+}
+----------------------------------
+// CONSOLE 
```

Comment:
+1 it might just be simpler to have them add it when they edit the elasticsearch.yml before?

Reply:
> +1 it might just be simpler to have them add it when they edit the elasticsearch.yml before?

yes, the number of master-eligible nodes should be specified upfront in the yml file, which also makes the discussion around the `monitor` cluster privileges obsolete. With Zen2, the configuration of this will completely change (see https://github.com/elastic/elasticsearch/pull/34714) and `minimum_master_nodes` will be replaced by a new setting. As this PR will be backported to 6.x, it's easier to make that change (replacement of minimum_master_nodes by initial_master_nodes) as a follow-up.